src/Microsoft.ApplicationInsights/TelemetryClient.cs

```diff
@@ -22,7 +22,11 @@
     /// </summary>
     public sealed class TelemetryClient
     {
+#if NETSTANDARD1_3 || NETSTANDARD2_0
+        private const string VersionPrefix = "dotnetc:";
```

Comment:
Some BI reports / business telemetry may need to get corrected. Can you please email PM team about this change after it's in?

Reply:
Yes. https://github.com/Microsoft/ApplicationInsights-home Will update the SDKVersion here as well.